Source/WebKit/UIProcess/ios/WKActionSheetAssistant.mm

```diff
@@ -396,6 +396,9 @@ - (void)showImageSheet
             targetURL = alternateURL;
         auto elementBounds = _positionInformation->bounds;
         auto elementInfo = adoptNS([[_WKActivatedElementInfo alloc] _initWithType:_WKActivatedElementTypeImage URL:targetURL imageURL:imageURL location:_positionInformation->request.point title:_positionInformation->title ID:_positionInformation->idAttribute rect:elementBounds image:_positionInformation->image.get() imageMIMEType:_positionInformation->imageMIMEType userInfo:userInfo]);
+        if (alternateURL)
```

Comment:
I think this condition should be `!targetURL && alternateURL`, because if there is a `targetURL`, that is currently used (see line 395-396). I don't think we should change that behavior.

The property could also be renamed to `isUsingAlternateURLForImage` to better reflect when an "alternate URL" is used.